GlobalFrame: "map"
RobotFrame: "robot"

OdometryTopic: "robot/odometry"
VelocityTopic: "robot/target_velocity"

ControlHZ: 500 #100~1000
OdometryHZ: 100